tests/Application/templates/bundles/SyliusShopBundle/Checkout/SelectPayment/_choice.html.twig

```diff
@@ -0,0 +1,19 @@
+<div class="item" {{ sylius_test_html_attribute('payment-item') }}>
+    <div class="field">
+        <div class="ui radio checkbox" {{ sylius_test_html_attribute('payment-method-checkbox') }}>
+            {{ form_widget(form, sylius_test_form_attribute('payment-method-select')) }}
+        </div>
+    </div>
+    <div class="content">
+        <a class="header">{{ form_label(form, null, {'label_attr': {'data-test-payment-method-label': ''}}) }}</a>
+        {% if method.description is not null %}
+            <div class="description">
+                <p>{{ method.description }}</p>
+            </div>
+        {% endif %}
+
+        {% if method.gatewayConfig.factoryName == 'akawaka_soge_commerce' %}
```

Comment:
if you create a const in PHP for 'akawaka_soge_commerce' we will be able to use it here